### PR TITLE
[NEW] Rest users presence

### LINKED ISSUE #1023: Add users.actives endpoint to documentation
https://github.com/RocketChat/Rocket.Chat/pull/12800/files#diff-1d5c7afdaba4728789f33cfa02ec1515

### PR DESCRIPTION
**Pull request by MarcosSpessatto**
Closes https://github.com/RocketChat/docs/issues/1023
Waiting https://github.com/RocketChat/Rocket.Chat/pull/12800

### CODE REVIEW

**Review by sampaiodiego — APPROVED** [review]

File: _data/toc.yml [hunk 1]
```diff
@@ -435,6 +435,7 @@
         - read
         - unread
       - Users:
+        - active
```
Comment (geekgonecrazy) [inline _data/toc.yml]:
```suggestion
        - presence
```

File: contributing/documentation/documentation-map/README.md [hunk 2]
```diff
@@ -433,6 +433,7 @@ Here you can also find what articles are incomplete and missing.
             - read
             - unread
         - Users:
+            - active
```
Comment (geekgonecrazy) [inline contributing/documentation/documentation-map/README.md]:
```suggestion
            - presence
```

File: developer-guides/rest-api/README.md [hunk 3]
```diff
@@ -49,6 +49,7 @@ When calling a production Rocket.Chat server, ensure it is running via HTTPS and
 
 | Url                                           | Short Description                                               | Details Page                                 |
 | :-------------------------------------------- | :-------------------------------------------------------------- | :------------------------------------------- |
+| `/api/v1/users.active`                        | Gets all active users.                                          | [Link](users/active/)                        |
```
Comment (geekgonecrazy) [inline developer-guides/rest-api/README.md]:
```suggestion
| `/api/v1/users.presence`                        | Gets all connected users presence.                                          | [Link](users/presence/)                        |
```

File: developer-guides/rest-api/users/README.md [hunk 4]
```diff
@@ -2,6 +2,7 @@
 
 | Url | Short Description | Details Page |
 | :--- | :--- | :--- |
+| `/api/v1/users.active` | Gets all active users. | [Link](active/) |
```
Comment (geekgonecrazy) [inline developer-guides/rest-api/users/README.md]:
```suggestion
| `/api/v1/users.presence` | Gets all connected users presence. | [Link](presence/) |
```

File: developer-guides/rest-api/users/active/README.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
+# User Active
```
Comment (geekgonecrazy) [inline developer-guides/rest-api/users/active/README.md]:
```suggestion
# Users Presence
```

File: developer-guides/rest-api/users/active/README.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+Gets all active users.
```
Comment (geekgonecrazy) [inline developer-guides/rest-api/users/active/README.md]:
```suggestion
Gets all connected users presence.
```

File: developer-guides/rest-api/users/active/README.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+
+| URL | Requires Auth | HTTP Method |
+| :--- | :--- | :--- |
+| `/api/v1/users.active` | `yes` | `GET` |
```
Comment (geekgonecrazy) [inline developer-guides/rest-api/users/active/README.md]:
```suggestion
| `/api/v1/users.presence` | `yes` | `GET` |
```

File: developer-guides/rest-api/users/active/README.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+
+## Other Users Example Call
+
+```bash
+curl -H "X-Auth-Token: 9HqLlyZOugoStsXCUfD_0YdwnNnunAJF8V47U3QHXSq" \
+     -H "X-User-Id: aobEdbYhXfu5hkeqG" \
+     http://localhost:3000/api/v1/users.active
```
Comment (geekgonecrazy) [inline developer-guides/rest-api/users/active/README.md]:
```suggestion
     http://localhost:3000/api/v1/users.presence
```

File: developer-guides/rest-api/users/active/README.md [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+   ],
+   "success":true
+}
+```
+
+## Change Log
+
+| Version | Description |
+| :--- | :--- |
+| 0.73.0 | Added |
```
Comment (geekgonecrazy) [inline developer-guides/rest-api/users/active/README.md]:
```suggestion
| 1.1.0 | Added |
```

**Review by geekgonecrazy — CHANGES_REQUESTED** [review]

**Comment by geekgonecrazy** [comment]
Changed with new endpoint: https://github.com/RocketChat/Rocket.Chat/pull/14488

**Comment by MarcosSpessatto** [comment]
Wow, thanks for updating the docs @geekgonecrazy :1st_place_medal:

**Comment by sampaiodiego** [comment]
guys I added the `from` parameter description and some docs about the stream payload.

Please review as I'm not sure the wording. thanks

**Review by geekgonecrazy — APPROVED** [review]

**Comment by geekgonecrazy** [comment]
LGTM 👍

**Pull request merged**